data/pools/transactionPool.go

```diff
@@ -548,20 +534,9 @@ func (pool *TransactionPool) OnNewBlock(block bookkeeping.Block, delta ledgercor
 		stats = pool.recomputeBlockEvaluator(committedTxids, knownCommitted)
```

Comment:
the knownCommitted var is passed uninitialized

Reply:
I've actually removed it entirely in the latest change since it was only used for hint computation and only nodes with enableProcessBlockStats ever went into the loop that would initialize it.